scripts/third_party_size_check.py

```diff
@@ -28,47 +28,54 @@
 
 
 def _get_skip_files_list():
-    """This lists the directories which should be skipped."""
+    """This function returns the list of the files which are skipped when
+    Oppia is deployed to GAE.
+
+    Returns:
+        The list of skip_files directories.
+    """
     with open('./app.yaml', 'r') as app_yaml:
         try:
             app_yaml_dict = yaml.safe_load(app_yaml)
         except yaml.YAMLError as exc:
             print exc
-        skip_files = app_yaml_dict.get('skip_files')
+        skip_files_list = app_yaml_dict.get('skip_files')
 
-        skip_files = [os.getcwd() + '/' + skip_files_dir
-                      for skip_files_dir in skip_files]
+        skip_files_list = [os.getcwd() + '/' + skip_files_dir
+                           for skip_files_dir in skip_files_list]
 
-    return skip_files
+    return skip_files_list
 
 
-def _check_size_in_dir(skip_files, dir_path):
+def _check_size_in_dir(dir_path, skip_files_list):
     """Recursive method that checks the number of files inside the given
     directory.
+
     Args:
-         skip_files: str. The directory which files will not be counted.
          dir_path: str. The directory which files will be counted.
+         skip_files_list: list. The directories which files will not be counted.
```

Comment:
Please change the description to `The list of files which are to be skipped from the file count.`

Reply:
Done Thanks!